trading_calendars/weekday_calendar.py

```diff
@@ -0,0 +1,25 @@
+from datetime import time
+
+from trading_calendars import TradingCalendar
+
+
+class WeekdayCalendar(TradingCalendar):
+    """
+    A TradingCalendar for an exchange that is open every minute of every
+    weekday.
+    """
+    @property
+    def name(self):
+        return '24/5'
+
+    @property
+    def tz(self):
+        return 'UTC'
+
+    @property
+    def open_time(self):
+        return time(0)
```

Comment:
Hrm. The convention we have elsewhere in this repo is that the open time is the label for the first minute of trading in a calendar, which means that, e.g. the first minute of the NYSE calendar is 9:31, not 9:30, and the last minute is 4:00. It seems like the analogous thing here would be to have (0, 1) and (24, 0), but (24, 0) isn't valid... I'm not sure how much that matters in practice for these 24 hour calendars though.

Reply:
I see. FWIW the always_open calendar uses this pattern. I could look into finding out how to possibly change them both if that's worth it?

Reply:
I don't see a good way to fix this right now. I'd leave it for the time being.